quantize.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+"""Script to execute quantization on a given model."""
+
+import subprocess
+import argparse
+import sys
+import os
+
+
+def main():
+    """Parse the command line arguments and execute the script."""
+
+    parser = argparse.ArgumentParser(
+        prog='Quantization Script',
+        description='This script quantizes a model.'
+    )
+
+    parser.add_argument("models", nargs='+', dest='models')
```

Comment:
could use choices here (if we want to limit to the 7/13/30/65 models): https://docs.python.org/3/library/argparse.html#choices

something like:

```python
parser.add_argument("models", nargs='+', dest='models', choices=("7B", "13B", "30B", "65B"))
```

Reply:
Added.